### ISSUE TITLE
'electron27' have been found in the repositories

### ISSUE THREAD

**Issue by Elimelech** [SrWither/DiscordBSD#8]
in  freebsd 14.0 only has electron25-25.9.7. 

```
root@F3ja:/usr/home/luba # pkg install electron27
Updating FreeBSD repository catalogue...
FreeBSD repository is up to date.
All repositories are up to date.
pkg: No packages available to install matching 'electron27' have been found in the repositories
```

Can the program DiscordBSD  be installed using electron25-25.9.7?

**Comment by SrWither** [comment]
Install electron is necessary only if you want to build from source,
If you install discordbsd from the release, is already packaged with electron, That is to say that electron already comes with discordbsd so to speak, If you need electron27 you can install it here:
https://github.com/tagattie/FreeBSD-Electron/releases/tag/v27.1.2

**Comment by grahamperrin** [comment]
I cancelled today's attempt to build. 

electron29 stated at <https://github.com/SrWither/DiscordBSD?tab=readme-ov-file#build-dependencies>, the attempt to build shows "depends on executable: electron28". 

```text
% ./build.sh 
Need to elevate permissions to pack DiscordBSD. Do you want to execute the command with doas or sudo? (1 for doas, 2 for sudo)
2
electron.mk checked

up to date, audited 2 packages in 4s

1 high severity vulnerability

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
npm notice
npm notice New patch version of npm available! 10.8.2 -> 10.8.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.3
npm notice To update run: npm install -g npm@10.8.3
npm notice
grahamperrin's password:
===>  License BSD3CLAUSE accepted by the user
===>  License BSD3CLAUSE accepted by the user
===>   discordbsd- depends on file: /usr/local/sbin/pkg - found
===>   discordbsd- depends on package: npm-node20>0 - found
===>   Setting up node modules cache directory
===>   Prefetching and archiving node modules
npm warn deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
npm warn deprecated asar@3.2.0: Please use @electron/asar moving forward.  There is no API change, just a package name change

added 514 packages in 30s
npm notice
npm notice New patch version of npm available! 10.8.2 -> 10.8.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.3
npm notice To update run: npm install -g npm@10.8.3
npm notice
=> SrWither-DiscordBSD--main_GH0.tar.gz doesn't seem to exist in /usr/ports/distfiles/.
=> Attempting to fetch https://codeload.github.com/SrWither/DiscordBSD/tar.gz/main?dummy=/SrWither-DiscordBSD--main_GH0.tar.gz
fetch: https://codeload.github.com/SrWither/DiscordBSD/tar.gz/main?dummy=/SrWither-DiscordBSD--main_GH0.tar.gz: size of remote file is not known
SrWither-DiscordBSD--main_GH0.tar.gz                   632 kB 5832 kBps    00s
===> Fetching all distfiles required by discordbsd- for building
===>  License BSD3CLAUSE accepted by the user
===>   discordbsd- depends on file: /usr/local/sbin/pkg - found
===>   discordbsd- depends on package: npm-node20>0 - found
===> Fetching all distfiles required by discordbsd- for building
===>  Extracting for discordbsd-
=> SHA256 Checksum OK for discordbsd--node-modules.tar.gz.
=> SHA256 Checksum OK for SrWither-DiscordBSD--main_GH0.tar.gz.
===>   Copying package.json and package-lock.json to /usr/home/grahamperrin/dev/DiscordBSD/freebsd/work/DiscordBSD-main
===>   Moving prefetched node modules to /usr/home/grahamperrin/dev/DiscordBSD/freebsd/work/DiscordBSD-main
===>  Patching for discordbsd-
===>   Patching package.json
===>   discordbsd- depends on executable: node - found
===>   discordbsd- depends on executable: electron28 - not found


To build electron, you should have around 2GB of memory
and around 14 GB of free disk space.

===>   NOTICE:

This port is deprecated; you may wish to reconsider installing it:

EOLed upstream, switch to newer version.

It is scheduled to be removed on or after 2024-08-31.

===>  License BSD3CLAUSE LGPL21 MPL11 MIT accepted by the user
===>   electron28-28.3.3_1 depends on file: /usr/local/sbin/pkg - found
===>   electron28-28.3.3_1 depends on package: yarn-node18>0 - not found

===>  License BSD2CLAUSE accepted by the user
===>   yarn-node18-1.22.19 depends on file: /usr/local/sbin/pkg - found
===> Fetching all distfiles required by yarn-node18-1.22.19 for building
===>  Extracting for yarn-node18-1.22.19
=> SHA256 Checksum OK for yarn-v1.22.19.tar.gz.
===>  Patching for yarn-node18-1.22.19
===>  Applying FreeBSD patches for yarn-node18-1.22.19 from /usr/ports/www/yarn-node18/../yarn-node20/files
===>  Configuring for yarn-node18-1.22.19
===>  Staging for yarn-node18-1.22.19
===>   yarn-node18-1.22.19 depends on executable: node - found
===>   Generating temporary packing list
/bin/mkdir -p /usr/ports/www/yarn-node18/work/stage/usr/local/lib/node_modules/yarn
(cd /usr/ports/www/yarn-node18/work/yarn-v1.22.19/ && /bin/sh -c '(/usr/bin/find -Ed $1 $3 | /usr/bin/cpio -dumpl $2 >/dev/null 2>&1) &&  /usr/bin/find -Ed $1 $3 \(   -type d -exec /bin/sh -c '\''cd '\''$2'\'' && chmod 755 "$@"'\'' . {} +  -o -type f -exec /bin/sh -c '\''cd '\''$2'\'' && chmod 0644 "$@"'\'' . {} + \)' COPYTREE_SHARE . /usr/ports/www/yarn-node18/work/stage/usr/local/lib/node_modules/yarn)
install -l rs /usr/ports/www/yarn-node18/work/stage/usr/local/lib/node_modules/yarn/bin/yarn.js  /usr/ports/www/yarn-node18/work/stage/usr/local/bin/yarn.js
install -l rs /usr/ports/www/yarn-node18/work/stage/usr/local/lib/node_modules/yarn/bin/yarnpkg  /usr/ports/www/yarn-node18/work/stage/usr/local/bin/yarnpkg
install -l rs /usr/ports/www/yarn-node18/work/stage/usr/local/lib/node_modules/yarn/bin/yarn  /usr/ports/www/yarn-node18/work/stage/usr/local/bin/yarn
====> Compressing man pages (compress-man)
===>  Installing for yarn-node18-1.22.19

===>  yarn-node18-1.22.19 conflicts with installed package(s): 
      yarn-node20-1.22.19

      They install files into the same place.
      You may want to stop build with Ctrl + C.
^C*** Signal 2
*** Signal 2
*** Signal 2
*** Signal 2
*** Signal 2
grahamperrin's password:
sudo: a password is required
cp: freebsd/work/DiscordBSD-main/out/DiscordBSD-linux-x64: No such file or directory
Done!
% pkg iinfo electron
electron29-29.4.6
electron30-30.4.0_1
electron31-31.4.0_1
% uname -aKU
FreeBSD mowa219-gjp4-zbook-freebsd 15.0-CURRENT FreeBSD 15.0-CURRENT main-n271938-3e76d05231b0 GENERIC-NODEBUG amd64 1500023 1500023
% 
```

**Comment by SrWither** [comment]
now discordbsd depends on electron30 which is in the repository
